### ISSUE TITLE
High loss and low SDR with Value Residual + Hyper Connections

### ISSUE THREAD

**Issue by lucellent** [lucidrains/BS-RoFormer#42]
I adapted Roman's code to work with the new changes, and I was getting 4-5 times higher loss than usual. I thought I did something wrong, so I used your exact code as is, but it turns out it's the exact same situation. 

Keep in mind I'm trying to fine-tune Kimebrley's model as I don't have the resources to train from scratch. Could this be the culprit? 
I also tried multiple other things, like adding grad clip, changing optimizers and learning rates, etc. but the issue remained.

Any insight is helpful, thank you!

**Comment by lucidrains** [comment]
@lucellent yes, if their model was trained using the old architecture, it cannot work with the new

**Comment by lucellent** [comment]
Ah pity, but makes sense. Thanks for confirming!
This applies to both right? Both value residual and hyper connections
